spec/models/quote_spec.rb

```diff
@@ -4,11 +4,6 @@
   let(:quote) { FactoryGirl.create(:quote) }
   let(:user) { FactoryGirl.create(:user) }
 
-  it "requires a body" do
-    quote.body = nil
-    expect(quote).to be_invalid
-  end
-
```

Comment:
I think there is some value in testing validations, but I prefer to use: https://github.com/thoughtbot/shoulda-matchers so you can:

```
it { is_expected.to validate_presence_of(:body) }
```

Which reads better.